src/test/java/DeletePlayList.java

```diff
@@ -0,0 +1,30 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.safari.SafariDriver;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+public class DeletePlayList extends BaseTest {
+
+    @Test
+    public static void addSongToPlaylist() throws InterruptedException {
+        navigateToPage("https://bbb.testpro.io/");
+
+        // Login to the Homepage Koel
+        provideEmail("janezelenova@gmail.com");
+        providePassword("Floridaliving2023$");
+        clickSubmit();
+        Thread.sleep(2000);
+
+        selectPlaylisttoDelete();
+        removePlaylist();
+        cancel();
```

Comment:
Please, **add Assert in test**

   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

        Assert.assertEquals(newName, getUsersNameOnHeader());

        Assert.assertEquals(newName, getUsernameProfileField());

Reply:
Ok. Thanks a lot for your help! I fixed my Homework #17 & Homework#18 as well if you'd like to check it and grade it as well. Both tests passed 😊